### PR TITLE
[Improvement] Improve docs for relasing 0.7.0

### PR DESCRIPTION
**Pull request by zhoujinsong**
## Why are the changes needed?

Fix some issues of docs for releasing 0.7.0

## Brief change log

- Fix some doc issues

## How was this patch tested?

- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] Run test locally before making a pull request

## Documentation

- Does this pull request introduce a new feature? (no)
- If yes, how is the feature documented? (not applicable )

### CODE REVIEW

File: docs/admin-guides/deployment.md [hunk 1]
```diff
@@ -172,20 +140,33 @@ AMS provides implementations of `LocalContainer` and `FlinkContainer` by default
 ```yaml
 containers:
   - name: localContainer
-    container-impl: org.apache.amoro.optimizer.LocalOptimizerContainer
+    container-impl: org.apache.amoro.server.optimizer.LocalOptimizerContainer
```
Comment (baiyangtx) [inline docs/admin-guides/deployment.md:143]:
```suggestion
    container-impl: org.apache.amoro.server.manager.LocalOptimizerContainer
```

File: docs/admin-guides/deployment.md [hunk 2]
```diff
@@ -266,26 +257,13 @@ The following JVM options could be set in `${AMORO_CONF_DIR}/jvm.properties`.
...
-
-
 ## Start AMS
 
 Enter the directory amoro-x.y.z and execute bin/ams.sh start to start AMS.
 
 ```shell
-cd amoro-x.y.z
-bin/ams.sh start
+$ cd amoro-x.y.z
```
Comment (tcodehuber) [inline docs/admin-guides/deployment.md:265]: apache-amoro-x.y.z-bin

File: docs/admin-guides/managing-catalogs.md [hunk 3]
```diff
@@ -59,15 +59,10 @@ Common properties include:
...
-
-- If you want to use it in conjunction with HMS, choose `External Catalog` for the `Type` and `Hive Metastore` for the `Metastore`, and choose the table format based on your needs, Mixed-Hive or Iceberg.
-- If you want to use Mixed-Iceberg provided by Amoro, choose `Internal Catalog` for the `Type` and `Mixed-Iceberg` for the table format.
-
 ## Delete catalog
 When a user needs to delete a Catalog, they can go to the details page of the Catalog and click the Remove button at the bottom of the page to perform the deletion.
 
 {{< hint info >}}
-Before deleting a Catalog, AMS will verify whether there is metadata for tables under that Catalog.
+Before deleting a internal catalog, AMS will verify whether there is metadata for tables under that Catalog.
```
Comment (tcodehuber) [inline docs/admin-guides/managing-catalogs.md:66]: an internal catalog

File: docs/admin-guides/deployment.md (hunk 2 above)
Comment (zhoujinsong) [reply docs/admin-guides/deployment.md:265]: The extracted folder name is still `amoro-x.y.z`, reference:https://github.com/apache/amoro/blob/master/amoro-ams/dist/src/main/assemblies/bin.xml#L28

File: docs/admin-guides/managing-catalogs.md (hunk 3 above)
Comment (zhoujinsong) [reply docs/admin-guides/managing-catalogs.md]: Fixed.

**Comment by zhoujinsong** [comment]
I have made some changes due to the review suggestion, PTAL. @tcodehuber @baiyangtx

**Review by tcodehuber — APPROVED** [review]
LGTM

**Pull request merged**